packages/expo-modules-core/ios/JSI/EXJavaScriptRuntime.mm

```diff
@@ -247,6 +258,14 @@ - (nonnull EXJavaScriptValue *)evaluateScript:(nonnull NSString *)scriptSource
 
 - (void)schedule:(nonnull JSRuntimeExecutionBlock)block priority:(int)priority
 {
+  if (_runtimeScheduler) {
+    auto schedulerPriority = static_cast<facebook::react::SchedulerPriority>(priority);
+    auto callback = [block](jsi::Runtime &) {
+      block();
+    };
+    _runtimeScheduler->scheduleTask(schedulerPriority, std::move(callback));
+    return;
+  }
```

Comment:
In our native tests, the scheduler won't be available, right? Then we still need to use `jsCallInvoker->invokeAsync`? Might be worth adding a comment to clarify when and which one is used.
I think we can also drop support for RN < 75 now, see below 👇

Reply:
The scheduler should always be available. I left the `jsCallInvoker` as a fallback because `RuntimeSchedulerBinding::getBinding` can return a nullptr so seemed safest to do that. I'm not sure under what circumstances it would not be available. Looks like we don't have any tests the touch this code path